File: petManage_app/src/app/model/cliente.ts
export class Cliente {
    idCliente: number;
    nome: string;
    cpf: string;
    endereco: string;
    email: string;
    telefone: string;

    constructor() {
        this.idCliente = 0;
        this.nome = "";
        this.cpf = "";
        this.endereco = "";
        this.email = "";
        this.telefone = "";
    }
}

File: petManage_api/src/main/java/dot/petstore/model/Cliente.java
package dot.petstore.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Cliente {
    private int idCliente;
    private String nome;
    private String cpf;
    private String endereco;
    private String email;
    private String telefone;
}